example.py

```diff
@@ -0,0 +1,12 @@
+from phonon.connections import connect
+connect(hosts=['localhost'])
+
+from phonon.fields import *
+from phonon.model import Model
+
+class Foo(Model):
+    a = SumField()
```

Comment:
Maybe add a little more context about what this example file does? I couldn't tell if it mattered or not